pom.xml

```diff
@@ -19,41 +19,41 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <ramlfiles_path>${basedir}/ramls</ramlfiles_path>
     <ramlfiles_util_path>${basedir}/raml-util</ramlfiles_util_path>
-    <raml-module-builder.version>35.1.0</raml-module-builder.version>
+    <raml-module-builder.version>35.2.0</raml-module-builder.version>
     <java.version>17</java.version>
 
     <!--Dependency Management Properties-->
-    <vertx.version>4.4.5</vertx.version>
+    <vertx.version>4.5.4</vertx.version>
     <kafkaclients.version>3.3.2</kafkaclients.version>
-    <lombok.version>1.18.24</lombok.version>
+    <lombok.version>1.18.30</lombok.version>
     <postgres.image>postgres:12-alpine</postgres.image>
-    <log4j.version>2.20.0</log4j.version>
+    <log4j.version>2.23.0</log4j.version>
 
     <!--Dependency properties-->
     <streamx.version>0.8.1</streamx.version>
-    <rest-assured.version>5.3.2</rest-assured.version>
-    <aspectj.version>1.9.20.1</aspectj.version>
-    <aspectj-maven-plugin.version>1.13.1</aspectj-maven-plugin.version>
+    <rest-assured.version>5.4.0</rest-assured.version>
+    <aspectj.version>1.9.21.1</aspectj.version>
+    <aspectj-maven-plugin.version>1.14</aspectj-maven-plugin.version>
     <common-lang.version>2.6</common-lang.version>
     <jackson-bom.version>2.15.2</jackson-bom.version>
     <folio-di-support.version>2.0.1</folio-di-support.version>
     <mod-configuration-client.version>5.9.2</mod-configuration-client.version>
 
     <!--Maven plugin properties-->
-    <exec-maven-plugin.version>3.1.0</exec-maven-plugin.version>
+    <exec-maven-plugin.version>3.2.0</exec-maven-plugin.version>
     <maven-resources-plugin.version>3.3.1</maven-resources-plugin.version>
-    <maven-compiler-plugin.version>3.11.0</maven-compiler-plugin.version>
+    <maven-compiler-plugin.version>3.12.1</maven-compiler-plugin.version>
     <maven-antrun-plugin.version>3.1.0</maven-antrun-plugin.version>
-    <maven-shade-plugin.version>3.5.1</maven-shade-plugin.version>
+    <maven-shade-plugin.version>3.5.2</maven-shade-plugin.version>
     <maven-release-plugin.version>3.0.1</maven-release-plugin.version>
-    <maven-surefire-plugin.version>3.1.2</maven-surefire-plugin.version>
-    <build-helper-maven-plugin.version>3.4.0</build-helper-maven-plugin.version>
+    <maven-surefire-plugin.version>3.2.5</maven-surefire-plugin.version>
+    <build-helper-maven-plugin.version>3.5.0</build-helper-maven-plugin.version>
```

Comment:
lets also update mod-configuration-client to the latest released 5.10.0
Also please create new PR for edge-orders with updating edge-common to the latest edge-common v4.6.0